test/e2e-tests.sh

```diff
@@ -66,6 +119,14 @@ go_test_e2e -timeout=20m ./test || fail_test
 # Run these _after_ the integration tests b/c they don't quite work all the way
 # and they cause a lot of noise in the logs, making it harder to debug integration
 # test failures.
-./examples/smoke-test.sh || fail_test
```

Comment:
shouldn't `smoke-test.sh` script removed then ? :angel:

Reply:
yes, I actually removed it but didn't push that, will remove it now for real 😄